examples/scrollable/src/main.rs

```diff
@@ -1,44 +1,60 @@
-use iced::executor;
+use iced::widget::scrollable::{Scrollbar, Scroller};
 use iced::widget::{
-    button, column, container, horizontal_rule, progress_bar, radio,
-    scrollable, text, vertical_space, Row,
+    button, column, container, horizontal_space, progress_bar, radio, row,
+    scrollable, slider, text, vertical_space,
 };
+use iced::{executor, theme, Alignment, Color, Vector};
 use iced::{Application, Command, Element, Length, Settings, Theme};
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref SCROLLABLE_ID: scrollable::Id = scrollable::Id::unique();
+}
 
 pub fn main() -> iced::Result {
     ScrollableDemo::run(Settings::default())
 }
 
 struct ScrollableDemo {
-    theme: Theme,
-    variants: Vec<Variant>,
+    scrollable_direction: Direction,
+    scrollbar_width: u16,
+    scrollbar_margin: u16,
+    scroller_width: u16,
+    current_scroll_offset: Vector<f32>,
```

Comment:
I'm not sure using a `Vector` for storing the offsets is appropriate. I see offsets as specific points in the scrollable, rather than a length with a direction. In other words, they represent a position rather than a "movement".

Maybe we should use `Point` instead.